devnotes/payload_starting.md

```diff
@@ -191,6 +194,28 @@ pflash -P PAYLOAD -r /tmp/skiboot.bin
 tail -c +$((0x1001)) skiboot.bin | unxz > skiboot.lid
 ```
 
+### Building skiboot.lid
+
+   1. Clone the skiboot repository
+   ```
+   git clone https://git.raptorcs.com/git/talos-skiboot
```

Comment:
@IgorBagnucki Can we clarify which revision was used?

Reply:
I have added information, what revision should be checked out